#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_interactive_marker'

try:
    import imp
    imp.find_module(PACKAGE)
    from dynamic_reconfigure.parameter_generator_catkin import *;
except:
    import roslib; roslib.load_manifest(PACKAGE)
    from dynamic_reconfigure.parameter_generator import *;

from math import pi

gen = ParameterGenerator ()

gen.add("param0", double_t, 0, "param[0]", 0.5, 0.0, 1.0)
gen.add("param1", double_t, 0, "param[1]", 0.5, 0.0, 1.0)
gen.add("param2", double_t, 0, "param[2]", 0.5, 0.0, 1.0)
exit (gen.generate (PACKAGE, "jsk_interactive_marker", "PointCloudCropper"))
